### PR TITLE
docs: routing docs review. Stage 1

### PR DESCRIPTION
**Pull request by bandantonio**

### CODE REVIEW

File: docs/routing/route_configuration_options.md [hunk 1]
```diff
@@ -2,86 +2,108 @@
...
+    - Domain name must be without protocol.
+    - Empty `Domain Name` field means the main app domain.
+    - To add a new domain, go to the **Router domains** section in the sidebar.
+        
+        ??? tip "Add a new domain"
+            ![ILC registry domains menu](../assets/routes/domain-create.png)
 
-- You can add new domain in **Router domains** menu.
+!!! note ""
+    The application renders only one domain at the same time. To add one header to several domains, you need to create the same route several times specifying the required domain for each route.
```
Comment (StyleT) [inline docs/routing/route_configuration_options.md:20]: The application renders only one domain at the same time. 

This is unclear to me... Which application?

File: docs/routing/route_configuration_options.md [hunk 2]
```diff
@@ -2,86 +2,108 @@
 
 ## Route domains
 
-ILC will check the exact match of domain names, accordingly, `secure.example.com` is not equal to `example.com`, so you may need to add the necessary subdomains.
+ILC checks the exact match of domain names. It means that `secure.example.com` is not equal to `example.com`, and you may need to add the necessary subdomains to handle this case properly.
 
-ILC can handle requests coming from multiple domains, so that you don't need to roll out individual instances of the ILC for every domain, you just can use one instance for all of them.
+ILC can handle requests coming from multiple domains, so you can use a single ILC instance to handle them instead of rolling out individual instances for every domain.
```
Comment (StyleT) [inline docs/routing/route_configuration_options.md:7]: This should be the first sentence IMO

File: docs/routing/route_configuration_options.md [hunk 3]
```diff
@@ -2,86 +2,108 @@
...
-
-- You can find more information about domain [here](../multi-domains.md)
+More information about domains is available in the [Multi-domains page](../multi-domains.md).
 
 ## Route template
 
-**Important!** There must be at least one template in the routing chain.
-
-The template is needed to build the structure of our page. So if you don't create a template, ILC won't know exactly where to render the content and will show you an error instead of the desired page.
+Template is an HTML file that is used to build the structure of your page. Template is **mandatory**. If it is missing, ILC won't be able to render your content properly and will throw an error.
```
Comment (StyleT) [inline docs/routing/route_configuration_options.md:35]: Template is not a mandatory option

File: docs/routing/route_configuration_options.md [hunk 4]
```diff
@@ -2,86 +2,108 @@
...
 
-![ILC registry template menu](../assets/routes/template-create.png) 
+![ILC registry template menu](../assets/routes/template-create.png)
 
 ## Route metadata
 
-Currently only one option is supported - `protected` with values `true` or `false`.
+Metadata is used to determine whether the page should be protected. If yes, access to the protected page will be granted only after the user fulfills the required conditions.
+
+### Supported options
```
Comment (StyleT) [inline docs/routing/route_configuration_options.md:50]: Actually this info shouldn't be here. As ILC itself doesn't interpret metadata in any way. It's done solely by plugins

**Review by StyleT — CHANGES_REQUESTED** [review]

File: docs/routing/route_configuration_options.md (hunk 3 above)
Comment (bandantonio) [reply docs/routing/route_configuration_options.md:35]: What about the very first template? It says, that ILC won't be able to properly render the content. I believe that template is something that you need at least once.

File: docs/routing/route_configuration_options.md (hunk 1 above)
Comment (bandantonio) [reply docs/routing/route_configuration_options.md:20]: The original sentence was kept here, so the information is available in the documentation as well.

Screen Shot 2022-02-04 at 10 41 00

Screen Shot 2022-02-04 at 10 43 42

I believe this aspect wasn't spot during the initial review. @DammneD ^^ FYI

Comment (bandantonio) [reply docs/routing/route_configuration_options.md:20]: Clarified with @DammneD

File: docs/routing/route_configuration_options.md (hunk 3 above)
Comment (bandantonio) [reply docs/routing/route_configuration_options.md:35]: Clarified with @DammneD

File: docs/routing/route_configuration_options.md (hunk 4 above)
Comment (bandantonio) [reply docs/routing/route_configuration_options.md:53]: Discussed with @DammneD

**Review by StyleT — APPROVED** [review]

**Review by nightnei — APPROVED** [review]

**Pull request merged**